pom.xml

```diff
@@ -132,6 +137,16 @@
         </configuration>
       </plugin>
     </plugins>
+
+    <resources>
```

Comment:
Strange that you need to do this, `src/test/resources` should already by on the Maven test classpath. I'm a bit worried that if you do this it will end up in the Hiveberg jar file.

Reply:
Yeah, this was something I tried to fix it for IntelliJ, it seems very temperamental with when it wants to find the properties file and when it doesn't, and this was the only fix i tried that worked - i can probably revert and just have this as something I add locally when I need logging